pkg/model/stack_test.go

```diff
@@ -199,7 +199,7 @@ services:
       - OPTION_A=Cats
       - OPTION_B=Dogs
       - EMPTY_VAR=
-      - $PWD
+      - PWD
```

Comment:
Should the other case also work? I mean, if you specify only the `$PWD`.

I wouldn't like to break some existing case and this is a test case that is changing